Makefile

```diff
@@ -129,10 +137,62 @@ lint/makefile: checkmake
 	@echo "Linting Makefile..."
 	checkmake --config .checkmake Makefile
 
-lint/license: ## Check license headers
-lint/license: go-license
+# License header targets
+
+.PHONY: license-header-check
+license-header-check: ## Check license headers in source files
 	@echo "Checking license headers..."
-	go-license --config=license.yml --verify .
+	@licRes=$$(for pattern in $(LICENSE_FILE_PATTERNS); do \
```

Comment:
I know I'm the first on to introduce complete makefile actions but this is getting out of hand.

Let's create a `scripts` directory and add a `check-license-header.sh` in there with an additional `--fix` flag.

And add modify to make actions to be simpler.

Reply:
I think that can wait. However, moving these to a script file already would be an improvement because we can benefit from some additional shell formatters and linters. And also we don't need to make sure the `\` correctly put.